File: C_Programs/Prime Checker.c
#include <stdio.h>
#include <stdbool.h>
void main()
{
    int n,a,p;
    a = 2;
    p = true;
    
    printf("Enter Your Number: ");
    scanf("%d",&n);
    
    while (n>a)
    {
        if (n%a==0)
        {
        p = false;
        break;
        }
        
    a++;
    }
    
    if(p==true)
    printf("%d is Prime",n);
    else
    printf("%d is Not-Prime",n);
}

File: Prime_checker.py
# Prime Checker

def Prime_Checker(n):
    a = 2
    p = True

    while n > a:
        if n % a == 0:
            p = False
            break
        a = a + 1
    return p

while True:
    n = int(input("Enter Your Number: "))
    #q = Prime_Checker(n)  # Call the Prime_Checker function and assign its return value to p

    if Prime_Checker(n):  # Call the function with the input n
        print("Prime")
    else:
        print("Not Prime")

    q = input("Do you want to repeat_(Y/N): ").lower()
    if q != "y":
        break

input("\nEnter to Close...")